src/client/types.ts

```diff
@@ -30,6 +30,12 @@ export type ClientRequest<S extends Schema> = {
         : {}
       : {}
   ) => URL
+} & {
+  $ws: S['$get'] extends { input: { json: '_websocket' | undefined } }
```

Comment:
It's better to use the `_websocket` as a constant because it's also used in line 17 in `src/helper/websocket/index.ts`.

And, I like `__websocket` rather than `_websocket` to avoid duplicating user input values as much as possible.

Reply:
I implemented it.